File: app/src/main/java/com/ben/bencustomerserver/views/EaseEmojiconMenuListener.java
package com.ben.bencustomerserver.views;


public interface EaseEmojiconMenuListener {
        /**
         * on emojicon clicked
         * @param emojicon
         */
        void onExpressionClicked(Object emojicon);
        /**
         * on delete image clicked
         */
        default void onDeleteImageClicked() {}
}

File: app/src/main/java/com/ben/bencustomerserver/listener/EaseEmojiconMenuListener.kt
package com.ben.bencustomerserver.listener

interface EaseEmojiconMenuListener {
    /**
     * on emojicon clicked
     * @param emojicon
     */
    fun onExpressionClicked(emojicon: Any?)

    /**
     * on delete image clicked
     */
    fun onDeleteImageClicked() {}
}